lib/index.js

```diff
@@ -109,24 +110,22 @@ Parsely.prototype.track = function(track) {
   return this.trackDynamicEvent(track);
 };
 
+/**
+ * Parse.ly requires metadata be passed about the video for started AND paused events.
+ * Since our video spec does not specify passing any metadata about the video, we need
+ * to store it from the videoContentStarted event as a global variable that the 
+ * videoPlaybackPaused event can access.
+ */
+
+var CURRENT_VIDEO_METADATA = {};
```

Comment:
this won't be scoped to `window` is it? Either way, I think it's better to keep this as a interval property of the integration instance `this.CURRENT_VIDEO_METADATA`

Reply:
Nah all these are wrapped in closures. Otherwise we would have big problems with all these:

https://github.com/segment-integrations/analytics.js-integration-parsely/blob/master/lib/index.js#L7-L11